config/locales/devise.en.yml

```diff
@@ -8,10 +8,10 @@ en:
     failure:
       already_authenticated: You are already signed in.
       inactive: Your account is not activated yet.
-      invalid: Invalid %{authentication_keys} or password.
+      invalid: Unable to login.
```

Comment:
Initially I wondered if we needed these three distinct names, given that we want the same message in each case, but then I realized that these are built into devise (we provide the translations, but devise calls by these keys). 

I was also going to ask about the French versions of these, but there isn't a .fr.yml file, and the devise gem only has the English locale as well, so nothing to do there.